### ISSUE TITLE
Add support for darcy

### ISSUE THREAD

**Issue by Gitski** [openandroidinstaller-dev/openandroidinstaller#237]
**Device informations**
- Brand: Google
- Phone name: Pixel 5
- Device code (if unknown, leave blank): 
- Model (if several models exist): Pixel 5

**Device images**  
Please provide links to any available images of the device, such as official stock firmware or custom ROMs you would like to install.  https://www.xda-developers.com/how-to-download-android-14/#download-android-14-beta-for-google-pixel-devices

**Do you own the device and would be willing to test the installer?**
- [ Yes ]

**Additional context**  
Please provide any additional context or information that may be helpful in adding support for this device.  

I actively support the advancement of all forms of techniology where possible. Given this is a spare device I am willing to participate in any part of the development lifecycle to help improve the overall reliability, security and user experience for any given product,

**Comment by anon1892** [comment]
Pixel 5 code name is `redfin`, and is already supported and is working with OpenAndroidInstaller.
`darcy` seems to be nVidia Shield TV 5. Am I wrong ?

**Comment by Gitski** [comment]
I recently installed the Android 14 developer beta on the pixel 5, when I attempted to install OoenAndroidInstaller it failed to install sayiing Darcy is yet to be supported.

**Comment by tsterbak** [comment]
> I recently installed the Android 14 developer beta on the pixel 5, when I attempted to install OpenAndroidInstaller it failed to install sayiing Darcy is yet to be supported.

Well, thats interesting! Can you maybe upload the respective log-file so we can investigate?

**Comment by MagicLike** [comment]
> Pixel 5 code name is `redfin`, and is already supported and is working with OpenAndroidInstaller. `darcy` seems to be nVidia Shield TV 5. Am I wrong ?

true, `darcy` is the wrong one - see Google stock ROM page: https://developer.android.com/about/versions/14/download
